Add changelog for v10.2.0-rc.1 (#940)
mapbox/mapbox-maps-android: CHANGELOG.md

Mapbox welcomes participation and contributions from everyone.

# 10.2.0-rc.1 December 2, 2021

## Bug fixes 🐞

> * Fix suggestion message to replace deprecated `queryRenderFeatures` overloaded methods. ([#878](https://github.com/mapbox/mapbox-maps-android/pull/878))

Reviewer: ```suggestion
* Fix `replaceWith` template to replace deprecated `queryRenderFeatures` overloaded methods. ([#878](https://github.com/mapbox/mapbox-maps-android/pull/878))
```

Mapbox welcomes participation and contributions from everyone.

# 10.2.0-rc.1 December 2, 2021

## Bug fixes 🐞
* Fix `replaceWith` template to replace deprecated `queryRenderFeatures` overloaded methods. ([#878](https://github.com/mapbox/mapbox-maps-android/pull/878))
* Do not allow to use one `associatedFeatureId` with multiple view annotations. ([#896](https://github.com/mapbox/mapbox-maps-android/pull/896))
* Fix an issue where shove gesture was not detected when angle between touch points are not horizontal. ([#875](https://github.com/mapbox/mapbox-maps-android/pull/875))
* Fix gestures at high camera pitch near horizon line. ([#927](https://github.com/mapbox/mapbox-maps-android/pull/927), [#925](https://github.com/mapbox/mapbox-maps-android/pull/925))
* Fix fading tiles layout visibility issue on globe view projection zooming in/out. ([#925](https://github.com/mapbox/mapbox-maps-android/pull/925))

> * Add config to enable simultaneous zoom and rotate gesture. ([#885](https://github.com/mapbox/mapbox-maps-android/pull/885))

Reviewer 2: shall we mention the default behaviour is changed(fixed) as we now by default allow simultaneous zoom and rotate gesture?

Reviewer 2: ```suggestion
* Allow simultaneous zoom and rotate gesture by default and add `GesturesSettings#simultaneousRotateAndPinchToZoomEnabled` config option. ([#885](https://github.com/mapbox/mapbox-maps-android/pull/885))
```

Author: should we mention it in changelog or in release note?

Reviewer 2: I think we should

Mapbox welcomes participation and contributions from everyone.

# 10.2.0-rc.1 December 2, 2021

## Features ✨ and improvements 🏁

> * Fix suggestion message to replace deprecated `queryRenderFeatures` overloaded methods. ([#878](https://github.com/mapbox/mapbox-maps-android/pull/878))

Reviewer 3: move to bugs, it's not a feature
`Features ✨ and improvements 🏁` section is not needed

Mapbox welcomes participation and contributions from everyone.

# 10.2.0-rc.1 December 2, 2021

## Features ✨ and improvements 🏁

> * Cancellable queryRenderFeatures ([#878](https://github.com/mapbox/mapbox-maps-android/pull/878))

Reviewer 3: And move to bugs as cancelable QRF were already introduced before
```suggestion
* Fix suggestion message to replace deprecated `queryRenderFeatures` overloaded methods. ([#878](https://github.com/mapbox/mapbox-maps-android/pull/878))
```

Reviewer: Is it more a bugfix, nor an improvement / feature?

Reviewer: @kiryldz probably not a `suggestion message` but `replaceWith` template or similar name

## Bug fixes 🐞
* Do not allow to use one associatedFeatureId with multiple view annotations. ([#896](https://github.com/mapbox/mapbox-maps-android/pull/896))
* Fix an issue where shove gesture was not detected when angle between touch points are not horizontal. ([#875](https://github.com/mapbox/mapbox-maps-android/pull/875))
* Fix gestures at high camera pitch near horizon line. ([#927](https://github.com/mapbox/mapbox-maps-android/pull/927))

> * Reduce drag sensitivity around and above the horizon. ([#925](https://github.com/mapbox/mapbox-maps-android/pull/925))

Reviewer 3: I guess it's not needed as we have #927 
@pengdev could you please advise here?

Reviewer 2: yes, it's the gl-native part fix of #927, maybe we can merge this entry with #927

Reviewer 2: something like 

* Fix gestures at high camera pitch near horizon line. ([#925](https://github.com/mapbox/mapbox-maps-android/pull/925), [#927](https://github.com/mapbox/mapbox-maps-android/pull/927))

> * Fix Fading tiles layout visibility issue on globe view projection zooming in/out. ([#925](https://github.com/mapbox/mapbox-maps-android/pull/925))

Reviewer 3: ```suggestion
* Fix fading tiles layout visibility issue on globe view projection zooming in/out. ([#925](https://github.com/mapbox/mapbox-maps-android/pull/925))
```
